portal-api/src/v1/middleware/utilisation-report-file-upload.js

```diff
@@ -0,0 +1,23 @@
+const multer = require('multer');
+const { FILE_UPLOAD } = require('../../constants');
+
+const fileFilter = (req, file, cb) => {
+  const fileExtension = file.originalname.match(/\.[^.]*$/g);
```

Comment:
Node has a built in function to allow us to get the file extension, so this can be
```
const path = require('path');
...
const fileExtension = path.extname(file.originalname);
```